### ISSUE TITLE
Add checks for OpenJDK Style §3.10 - Variable Declarations

### ISSUE THREAD

**Issue by lithops-zty** [checkstyle/checkstyle#19641]
Parent issue: #19604 

Description
---
Add Checkstyle checks for all rules under §3.10 - Variable Declarations of the OpenJDK Java Style Guidelines v6.

Rules Covered
---
From [§3.10 - Variable Declarations](https://cr.openjdk.org/~alundblad/styleguide/index-v6.html#toc-variable-declarations):
1. One variable per declaration (and at most one declaration per line) - `MultipleVariableDeclarations Check`
2. Square brackets of arrays should be at the type (String[] args) and not on the variable (String args[]). - `ArrayTypeStyle Check` with `javaStyle` set to `true` (default)
3. Declare a local variable right before it is first used, and initialize it as close to the declaration as possible. - `VariableDeclarationUsageDistance Check`

Checks to add to `openjdk_checks.xml`
---
### TreeWalker-level
```xml
<module name="TreeWalker">
  <module name="MultipleVariableDeclarations"/>
  <module name="ArrayTypeStyle"/>
  <module name="VariableDeclarationUsageDistance"/>
</module>
```

Progress Tracker
---
| Check | Rule | PR |
|---|---|---|
| `MultipleVariableDeclarations` | One variable per declaration | |
| `ArrayTypeStyle` | Java-style array declaration | |
| `VariableDeclarationUsageDistance` | Local variable declaration right before first usage | |

**Comment by vivek-0509** [comment]
@lithops-zty looks good to me please make PR

**Comment by lithops-zty** [comment]
@romani Can you help approve this issue if there's no problem?

**Comment by romani** [comment]
> Square brackets

are merged.

**Comment by lithops-zty** [comment]
@romani @Aman-Baliyan All three checks for Section 3.10 are completed.

**Comment by romani** [comment]
All is covered by @lithops-zty .
thanks a lot.
